### ISSUE TITLE
Rule 'use-default' to check default case in switch statement

### ISSUE THREAD

**Issue by roadhump** [eslint/eslint#787]
It would be nice to have rule to check presence of `default` case in switch statement

Valid

``` js
/*eslint use-default:2 */
switch (a) {
    case 1:
        // ...
        break;
    default:
        // ...
        break;
}
```

Invalid

``` js
/*eslint use-default:2 */
switch (a) {
    case 1:
        // ...
        break;
    case 2:
        // ...
        break;
}

// Error:  Expected a default case.
```

It should be possible to use `no default` comment to indicate and skip check

``` js
/*eslint use-default:2 */
switch (a) {
    case 1:
        // ...
        break;
    case 2:
        // ...
        break;

    /* no default */
}
```
